api/logs/permissions.py

```diff
@@ -12,8 +12,14 @@ class ContributorOrPublicForLogs(permissions.BasePermission):
     def has_object_permission(self, request, view, obj):
         assert isinstance(obj, (NodeLog)), 'obj must be a NodeLog, got {}'.format(obj)
 
-        for node_id in obj._backrefs['logged']['node']['logs']:
-            node = get_object_or_error(Node, node_id, display_name='node')
-            if ContributorOrPublic().has_object_permission(request, view, node):
+        if obj._backrefs.get('logged'):
```

Comment:
For example, it looks like the recent activity widget was removed from the home page for now, so all of my event tracking of that module needs to go.  But I will wait until I see what else is changing.